database/setup-db.sh

```diff
@@ -171,7 +171,7 @@ create_other() {
 
 grant() {
   # psql -c "ALTER DATABASE \"$RCT_DB_NAME\" OWNER TO \"$RCT_DB_USERNAME\""
-  psql -c "GRANT ALL PRIVILEGES ON DATABASE \"$RCT_DB_NAME\" TO \"$RCT_DB_USERNAME\""
+  # psql -c "GRANT ALL PRIVILEGES ON DATABASE \"$RCT_DB_NAME\" TO \"$RCT_DB_USERNAME\""
```

Comment:
Please never commit commented out code. This need to be deleted, we can find a solution together if you want, and create a PR to merge before this one.

Reply:
commented out code delete